js/summary.go

```diff
@@ -168,8 +168,9 @@ func summarizeMetricsToObject(data *lib.Summary, options lib.Options) map[string
 		"noColor":           data.NoColor, // TODO: move to the (runtime) options
 	}
 	m["state"] = map[string]interface{}{
-		"isStdOutTTY": data.UIState.IsStdOutTTY,
-		"isStdErrTTY": data.UIState.IsStdErrTTY,
+		"isStdOutTTY":     data.UIState.IsStdOutTTY,
+		"isStdErrTTY":     data.UIState.IsStdErrTTY,
+		"testRunDuration": int64(data.TestRunDuration.Round(time.Second) / time.Second),
```

Comment:
maybe provide it as is or in milliseconds? I don't think it will be used much but if someone has 300ms test run, having It reported as taking 0s is only *technically* correct ;)

Reply:
:+1: I'll fix this, and I think I'll even rename it to `testRunDurationMs` and make it a float for maximum precision

Reply:
Resolved by the latest [force-push](https://github.com/k6io/k6/compare/25cd5281f4b120f1a179b569483c9a4278b7b248..c7e8a7b34824aa8619886c96129dbbbdd4d6c6b6)